include/nihstro/shader_bytecode.h

```diff
@@ -225,7 +225,7 @@ struct DestRegister {
 };
 
 struct OpCode {
-    enum class Id : uint32_t {
+    enum Id : uint32_t {
```

Comment:
How is this related to a JIT? (also, none of the nihstro code was fixed for this change.)